### ISSUE TITLE
Tests sometimes flake with: "user@1000.service: Failed with result 'protocol'."

### ISSUE THREAD

**Issue by miabbott** [openshift/os#691]
(copied from https://bugzilla.redhat.com/show_bug.cgi?id=1914362)

```
Dec 14 20:16:57.891063 systemd[1]: Starting User Manager for UID 1000...
Dec 14 20:16:57.906946 systemd[1897]: pam_unix(systemd-user:session): session opened for user core by (uid=0)
Dec 14 20:16:57.950027 systemd[1897]: Failed to fully start up daemon: Permission denied
Dec 14 20:16:57.951477 systemd[1899]: pam_unix(systemd-user:session): session closed for user core
Dec 14 20:16:57.957096 systemd[1]: user@1000.service: Failed with result 'protocol'.
Dec 14 20:16:57.959203 sshd[1893]: pam_systemd(sshd:session): Failed to create session: Start job for unit user@1000.service failed with 'failed'
Dec 14 20:16:57.957389 systemd[1]: Failed to start User Manager for UID 1000.
```

We've hit it again in the pipeline for `luks.sss.t2`.

Seems to happen mostly in `luks.sss.*` tests.

---

We worked around this by not failing on this error in https://github.com/coreos/coreos-assembler/pull/2261 but I think it is still good to have a record of the failure in an issue.

**Comment by miabbott** [comment]
A fix to `systemd` was released to the RHEL 8.4.z EUS stream today that we believed may fix this issue.

See https://bugzilla.redhat.com/show_bug.cgi?id=1990403 -> https://access.redhat.com/errata/RHBA-2022:0776

**Comment by cgwalters** [comment]
Haven't seen this recently, closing.
